### PR TITLE
Fix/bring safari view to front

### PR DESCRIPTION
**Pull request by clpolet**
- Show Safari view controller as a modal view

- Dismiss it while disabling or if update token succeeded

### CODE REVIEW

File: MobileCenterDistribute/MobileCenterDistribute/MSDistributePrivate.h [hunk 1]
```diff
@@ -43,6 +43,11 @@ static NSString *const kMSUpdateTokenKey = @"MSUpdateToken";
 @interface MSDistribute ()
 
 /**
+ * View controller presenting the `SFSafariViewController`.
+ */
+@property(nonatomic) UIViewController *safariHostingViewController;
```
Comment (jaelim-ms) [inline MobileCenterDistribute/MobileCenterDistribute/MSDistributePrivate.h]: Any reason that you keep this as a property? Opening a browser is just one time thing, you can dispose it after it is done.

Comment (clpolet) [inline MobileCenterDistribute/MobileCenterDistribute/MSDistributePrivate.h]: We have to automatically dispose it later for instance if the module is disabled or if the openURL method is called with the right token. To do so we need to access the hosting view controller hence the property :)

File: MobileCenterDistribute/MobileCenterDistribute/Support/MobileCenterDistribute.xcconfig [hunk 2]
```diff
@@ -1,3 +1,3 @@
 #include "../../../Global.xcconfig"
 
-OTHER_LDFLAGS=$(inherited) -framework Foundation -framework UIKit -framework CoreTelephony -framework Security
+OTHER_LDFLAGS=$(inherited) -framework Foundation -framework UIKit -framework CoreTelephony
```
Comment (TroubleMakerBen) [inline MobileCenterDistribute/MobileCenterDistribute/Support/MobileCenterDistribute.xcconfig]: AFAIK this needs to include SafariServices, too. (It's added in the podspec but not here)

Comment (clpolet) [inline MobileCenterDistribute/MobileCenterDistribute/Support/MobileCenterDistribute.xcconfig]: Right

**Pull request merged**